pkg/nsx/services/subnetport/builder_test.go

```diff
@@ -190,6 +190,170 @@ func TestBuildSubnetPort(t *testing.T) {
 			},
 			expectedError: nil,
 		},
+		{
+			// DHCP/DHCPRelay - Y; StaticIPAllocation Enabled - N;
```

Comment:
It looks that the comments in L194 and L272 are not accurate. BTW when I wrote the cases with "xxx -Y/N", my original intention was use these flags to prevent missing cases...

Reply:
Thanks for reminder. Update the comments and slightly reorder the cases to make it clearer.